django-stubs/dispatch/dispatcher.pyi

```diff
@@ -19,7 +23,15 @@ class Signal:
     ) -> bool: ...
     def has_listeners(self, sender: Any = ...) -> bool: ...
     def send(self, sender: Any, **named: Any) -> list[tuple[Callable, str | None]]: ...
-    def send_robust(self, sender: Any, **named: Any) -> list[tuple[Callable, Exception | str]]: ...
+    def send_robust(self, sender: Any, **named: Any) -> list[tuple[Callable, Exception | Any]]: ...
     def _live_receivers(self, sender: Any) -> list[Callable]: ...
 
-def receiver(signal: list[Signal] | Signal, **kwargs: Any) -> Callable: ...
+_F = TypeVar("_F", bound=Callable[..., Any])
+
+def receiver(
+    signal: list[Signal] | tuple[Signal, ...] | Signal,
+    *,
+    sender: object | None = ...,
+    weak: bool = ...,
+    dispatch_uid: str | None = ...
```

Comment:
```
dispatch_uid
                An identifier used to uniquely identify a particular instance of
                a receiver. This will usually be a string, though it may be
                anything hashable.
```

So, let's use `Hashable` here.

Reply:
Updated it for `receiver`, `connect` and `disconnect`.